crates/core/src/kernel/models/actions.rs

```diff
@@ -281,15 +283,15 @@ impl From<String> for WriterFeatures {
 impl From<&str> for WriterFeatures {
     fn from(value: &str) -> Self {
         match value {
-            "appendOnly" => WriterFeatures::AppendOnly,
-            "invariants" => WriterFeatures::Invariants,
-            "checkConstraints" => WriterFeatures::CheckConstraints,
-            "changeDataFeed" => WriterFeatures::ChangeDataFeed,
+            "appendOnly" | "delta.appendOnly" => WriterFeatures::AppendOnly,
+            "invariants" | "delta.invariants" => WriterFeatures::Invariants,
+            "checkConstraints" | "delta.checkConstraints" => WriterFeatures::CheckConstraints,
+            "changeDataFeed" | "delta.enableChangeDataFeed" => WriterFeatures::ChangeDataFeed,
```

Comment:
@ion-elgreco why are these changes required? Are we not picking up table properties correctly right now? If so, is there a bug this is related to?

Reply:
@rtyler I am doing that so I can go from tableConfigKeys into tableFeatures. This is what spark-delta also does if your table protocol versions are 3,7 and you do set tblproperties "delta.enableChangeDataFeed" = true, it will additionally set the feature ChangeDataFeed into the protocol commit.

Reply:
hmm, just passing by, but this seems strange. these are the values written in the feature vector list, so this seems like a strange place to do this conversion as well, isnce it describes completely differnet fields, now? one is values in the feature vecotor field, andother one is keys in the table properties, which should be handled in the table config?

Reply:
I added the table config keys here as well for quick parsing to table features, see my comment above.

In spark when your protocol is at 3,7 it will auto convert the configs into table features so that's basically what I am doing here as well.